var/spack/repos/builtin/packages/reditools/REDItoolDenovo.py.patch

```diff
@@ -0,0 +1,11 @@
+--- a/main/REDItoolDenovo.py	2020-07-16 17:35:46.008030346 -0500
++++ b/main/REDItoolDenovo.py	2020-07-16 17:38:39.700035490 -0500
+@@ -768,7 +768,7 @@
+ for j in ridxinfo.split('\n'): #MOD
+ 	l=(j.strip()).split('\t')
+ 	if l[0] in ['*', '']: continue  #MOD
+- 	if int(l[2])+int(l[3]) > 0: rrefs[l[0]]=int(l[1])
++	if int(l[2])+int(l[3]) > 0: rrefs[l[0]]=int(l[1])
```

Comment:
I must be going blind, what is the change here? Tabs to spaces?

Reply:
Yes, it was actually mixed tabs and spaces and that caused parsing to fail.